src/bidiMapper/BidiServer.ts

```diff
@@ -19,11 +19,11 @@ import {EventEmitter} from '../utils/EventEmitter.js';
 import {LogType, LoggerFn} from '../utils/log.js';
 import type {Message} from '../protocol/protocol.js';
 import {ProcessingQueue} from '../utils/processingQueue.js';
+import {CdpConnection} from '../cdp/cdpConnection.js';
```

Comment:
the CDP implementation is not used in bidiMapper (it only needs the interface). Let's not import the implementation since it won't be needed by all consumers.

Reply:
Done